cas-server-webapp/pom.xml

```diff
@@ -79,6 +79,11 @@
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-web</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.pac4j</groupId>
+      <artifactId>pac4j-http</artifactId>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
The dependencies are very limited (at least for this submodule): slf4j, commons-codec, commons-lang3 and guava. I'll check it doesn't conflict with the ones from the CAS server (after resolving this issue: https://github.com/pac4j/pac4j/issues/317).